rero_ils/modules/holdings/cli.py

```diff
@@ -45,7 +45,7 @@ def get_document_pid_by_rero_number(rero_control_number):
         .filter('term', identifiedBy__value=rero_control_number).source(
             ['pid']).scan()
     documents = [document.pid for document in es_documents]
-    return next(iter(documents or []), None)
+    return documents[0] if documents else None
```

Comment:
It would be nice to not get all documents here, but as we are supposed to get only one result it is a nice to have.